MH4U Dex/CoreDataController.m

```diff
@@ -0,0 +1,446 @@
+//
+//  CoreDataController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/6/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "CoreDataController.h"
+#import <NSManagedObjectContext+Hydrate.h>
+#import "MonsterDrop.h"
+#import "AreaDrop.h"
+#import "Item.h"
+#import "Monster.h"
+#import "DamageZone.h"
+#import "Region.h"
+#import "Area.h"
+
+@interface CoreDataController ()
+
+@property (readwrite, strong, nonatomic) NSManagedObjectContext *managedObjectContext;
+@property (readwrite, strong, nonatomic) NSManagedObjectModel *managedObjectModel;
+@property (readwrite, strong, nonatomic) NSPersistentStoreCoordinator *persistentStoreCoordinator;
+
+@end
+
+@implementation CoreDataController
+
+NSString *const JSON = @"json";
```

Comment:
Constants that are only used within the `.m` file should be declared `static`; if the constant is used outside the `.m` file, it needs to be exported (`FOUNDATION_EXPORT`) in the corresponding header and the name needs to be prefixed with at least the project's three-letter prefix to avoid naming collisions.